llvm/docs/GlobalISel/GenericOpcode.rst

```diff
@@ -753,6 +753,22 @@ The type of the operand must be equal to or larger than the vector element
 type. If the operand is larger than the vector element type, the scalar is
 implicitly truncated to the vector element type.
 
+G_STEP_VECTOR
+^^^^^^^^^^^^^
+
+Create a scalable vector where all lanes are linear sequences starting at 0
+with a given unsigned step.
+
+The type of the operand must be equal to the vector element type.
```

Comment:
The equivalent node for SelectionDAG also specifies the wrapping behavior.

Should also document the restriction on scalable vector dest type that the verifier is enforcing.